wp-cxense.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Plugin Name: WP cXense
- * Version: 2.1.2
+ * Version: 2.1.3
```

Comment:
@jonastanz as your code is not a bugfix, but adding new, or modifying existing functionality, I would suggest you change the version number to 2.2.0 😄

Reply:
Changing to use a non-deprecated method, hmm not sure. What do the judges say? :)